config.example.toml

```diff
@@ -17,6 +17,11 @@ what = "fooo service"
 Repos = ['synyx/tuwat', 'synyx/buchungsstreber']
 Tag = 'gh'
 
+# This adds example alerts in every flavor, useful for development
+# Should be disabled in production use
+#[[example]]
+#Tag = 'demo'
+
```

Comment:
```suggestion
# This adds example alerts in every flavor, useful for development
# Should be disabled in production use
[[example]]
Tag = 'demo'

```

I would even make it default, so that the example is runnable just like that, even if github as rate-limited the user.

Reply:
I added a commit, making the example connector the default choice, with an additional hint that the provided github config does run out of the box, but will run into rate limits

Reply:
sehr cool!